services/web/server/src/simcore_service_webserver/exporter/formatters/models.py

```diff
@@ -97,6 +98,9 @@ def _validate_version(cls, v):
         return str(v)
 
 
+_MIGRATION_FLAGS = dict(pre=True, always=True)
```

Comment:
why put it here, we have these flags in the validators usually. Is this because it will be shared by multiple validators?

Reply:
moved to where it is used. and yes this needs to be shared by multiple migration validators